src/components/Footer/styles.ts

```diff
@@ -8,9 +8,10 @@ export const Wrapper = styled(Box)<BoxProps>`
   display: flex;
   align-items: center;
   justify-content: space-between;
-  background: ${theme.palette.loginColors?.pageWrapperBg};
-  border-top: 1px solid ${theme.palette.loginColors.border};
   padding: 20px 50px;
+  font-family: ${theme.typography.h5?.fontFamily};
+  font-size: ${theme.typography.body2?.fontSize};
+  font-weight: ${theme.typography.subtitle1?.fontWeight || 500};
```

Comment:
when theme.typography.subtitle1.fontWeight will not have weight, it should be a constant, I dont understand why ? exist